src/html/htmlcell.cpp

```diff
@@ -191,9 +191,9 @@ void wxHtmlCell::Layout(int WXUNUSED(w))
 
 
 
-const wxHtmlCell* wxHtmlCell::Find(int WXUNUSED(condition), const void* WXUNUSED(param)) const
+const wxHtmlCell* wxHtmlCell::Find(int condition, const void* param) const
 {
-    return nullptr;
+    return condition == wxHTML_COND_ISANCHOR && !GetId().IsEmpty() && GetId() == (*((const wxString*)param)) ? this : nullptr;
```

Comment:
A minor comment, but I think it would be nice to have a small inline helper function hiding this ugly cast (which ought to be a `static_cast<wxString*>`...) and use it here and in a couple of other places where the same check is done.

Reply:
I changed it to `static_cast` here and elsewhere.